typescript.example/examples/minimal-typecheck.ts

```diff
@@ -1,15 +1,18 @@
-var a;
-
-function f(x: string) {
-  return x;
+class Vehicle {
+	wheels: number
 }
 
-f(a);
+class Car extends Vehicle { 
+   engine: string
+} 
+ 
+var c: Car;
+const x = c.wheels;
 
-interface X {}
+interface E {
+	engine: string,
+	wheels: number
+}
+var z: E = c;
```

Comment:
This line is impossible to check in nabl2 because checking scope 'equality' is very limited.
In this specific example, the type of `c` is `RECORD(s1)` where `s1` is a scope that contains only `engine` and has an imported scope from `Vehicle` that contains `wheels`. 

Since `E` has both fields in one scope, but `c` has the fields in a parent/sub-scope structure, the check that the scope in `E` is a subset of the one in `c` fails.